### PR TITLE
Redesigned `check required plugins` import step

### PR DESCRIPTION
**Pull request by mlytvyn**
Migrated to Kotlin DSL + common IDEA styles

Screenshot 2024-02-01 at 22 33 48

### CODE REVIEW

File: src/com/intellij/idea/plugin/hybris/project/utils/Plugin.kt [hunk 1]
```diff
@@ -1,7 +1,6 @@
 /*
- * This file is part of "SAP Commerce Developers Toolset" plugin for Intellij IDEA.
- * Copyright (C) 2014-2016 Alexander Bartash <AlexanderBartash@gmail.com>
- * Copyright (C) 2019-2023 EPAM Systems <hybrisideaplugin@epam.com> and contributors
+ * This file is part of "SAP Commerce Developers Toolset" plugin for IntelliJ IDEA.
+ * Copyright (C) 2019-2024 EPAM Systems <hybrisideaplugin@epam.com> and contributors
```
Comment (AlexanderBartash) [inline src/com/intellij/idea/plugin/hybris/project/utils/Plugin.kt:3]: @mlytvyn Did I ask you not to do this?

Comment (mlytvyn) [reply src/com/intellij/idea/plugin/hybris/project/utils/Plugin.kt:3]: @AlexanderBartash , this is new class, GitHub shows incorrect changes.

Comment (AlexanderBartash) [reply src/com/intellij/idea/plugin/hybris/project/utils/Plugin.kt:3]: Because you've copied code from another class.

Comment (mlytvyn) [reply src/com/intellij/idea/plugin/hybris/project/utils/Plugin.kt:3]: I didn't, new class was created and existing class was removed.

Comment (AlexanderBartash) [reply src/com/intellij/idea/plugin/hybris/project/utils/Plugin.kt:3]: I am not going to argue you. Next time I see it, you will get DMCA'ed, again.

Comment (mlytvyn) [reply src/com/intellij/idea/plugin/hybris/project/utils/Plugin.kt:3]: compare files, there is nothing similar, from name to content.

Comment (AlexanderBartash) [reply src/com/intellij/idea/plugin/hybris/project/utils/Plugin.kt:3]: Okay, you started it.
You did not write this https://github.com/epam/sap-commerce-intellij-idea-plugin/blob/main/src/com/intellij/idea/plugin/hybris/project/descriptors/HybrisProjectDescriptor.java
So what should I DMCA this repository?

Comment (mlytvyn) [reply src/com/intellij/idea/plugin/hybris/project/utils/Plugin.kt:3]: @AlexanderBartash , y're absolutely right, copyright for some classes was updated by mistake. Thank you for sharing a valid example, I'll update copyright for that file accordingly.

Comment (AlexanderBartash) [reply src/com/intellij/idea/plugin/hybris/project/utils/Plugin.kt:3]: That is just one example of many. It is your job to find them and fix if you want to keep having this repository.

**Pull request merged**